html/modules/custom/reliefweb_job_tagger/src/Plugin/QueueWorker/OchaAiJobTagTaggerWorker.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+namespace Drupal\reliefweb_job_tagger\Plugin\QueueWorker;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Drupal\ocha_ai_tag\Services\OchaAiTagTagger;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Extract text from a document file.
+ *
+ * @QueueWorker(
+ *   id = "reliefweb_job_tagger",
+ *   title = @Translation("Use AI to tag jobs"),
+ *   cron = {"time" = 30}
+ * )
+ */
+class OchaAiJobTagTaggerWorker extends QueueWorkerBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The tagger.
+   *
+   * @var \Drupal\ocha_ai_tag\Services\OchaAiTagTagger
+   */
+  protected $jobTagger;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, EntityTypeManagerInterface $entity_type_manager, OchaAiTagTagger $job_tagger) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->entityTypeManager = $entity_type_manager;
+    $this->jobTagger = $job_tagger;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('entity_type.manager'),
+      $container->get('ocha_ai_tag.tagger'),
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function processItem($data) {
+    $nid = $data->nid;
+
+    if (empty($nid)) {
+      \Drupal::logger('deubg')->notice('no nid');
```

Comment:
```suggestion
      \Drupal::logger('debug')->notice('no nid');
```

Other instances of this typo below.

Reply:
Removed